Cesium.CodeGen/Ir/BlockItems/FunctionDefinition.cs

```diff
@@ -186,7 +186,7 @@ public void EmitTo(IEmitScope scope)
         syntheticEntrypoint.Body.Variables.Add(exitCode);
 
         var instructions = syntheticEntrypoint.Body.Instructions;
-        var atExitLdLocExitCode = Instruction.Create(OpCodes.Ldloc_S, exitCode);
+        var atExitLdLocExitCode = Instruction.Create(OpCodes.Nop);
```

Comment:
Let's get rid of this variable and `ldloc` altogether? I think it's a remnant of an earlier implementation that was entering the `try` block in this function.